sets/en.json

```diff
@@ -2848,6 +2848,22 @@
       "logo": "https://images.pokemontcg.io/sv7/logo.png"
     }
   },
+  {
+    "id": "tcgp1",
+    "name": "Genetic Apex",
+    "series": "TCG Pocket",
+    "printedTotal": 226,
+    "total": 286,
+    "ptcgoCode": "A1"
+  },
```

Comment:
Hey man, just asking, is there a reason why you didn't include the `releaseDate` and `images` properties, that are present in other sets, for the Pocket sets? I'm planning on doing a small personal project related to TCG Pocket using this API, so I'm just wondering if these fields will be available.